### ISSUE TITLE
Go libs cannot be found

### ISSUE THREAD

**Issue by plynch76** [helm/helm-www#721]
When trying to "go get" helm libraries, the process fails because it cannot find 
helm.sh/helm/v3/pkg/chart/loader

**Comment by mattfarina** [comment]
@plynch76 Can you provide the `go get` command you are using? What response are you getting? I'm unable to replicate the problem.

**Comment by plynch76** [comment]
go get
go get -insecure
Both commands return:
unrecognized import path "helm.sh/helm/v3/pkg/chart/loader": reading https://helm.sh/helm/v3/pkg/chart/loader?go-get=1: 404 Not Found

**Comment by mattfarina** [comment]
@plynch76 What are you trying to `go get`?

For example, are you running `go get helm.sh/helm/v3` or are you doing something else? I'm not entirely sure of the context. For instance, in a Go project I work on I just ran `go get helm.sh/helm/v3/pkg/chart/loader` and it added the package without any issue.

**Comment by plynch76** [comment]
In the go project I am working on, I am running a go get
helm.sh/helm/v3/pkg/chart/loader and getting the 404 error. I just tried as
well and got the error message. It used to work until a few weeks ago, but
I sometimes have issues with go get and other packages but those cleared
up. This one has been unreachable for a while now.

**Comment by bacongobbler** [comment]
@plynch76 try the following:

```
go get helm.sh/helm/v3
go get helm.sh/helm/v3/pkg/chart/loader
```

It's also important to keep in mind that you should be using go modules instead of `go get`. Example shown below...

```
root@b2d6e16e7e6e:/go/src/helloworld# pwd
/go/src/helloworld
root@b2d6e16e7e6e:/go/src/helloworld# cat main.go
package main

import (
        "helm.sh/helm/v3/pkg/chart/loader"
)

func main() {}
root@b2d6e16e7e6e:/go/src/helloworld# ls ..
helloworld
root@b2d6e16e7e6e:/go/src/helloworld# go mod init
go: creating new go.mod: module helloworld
root@b2d6e16e7e6e:/go/src/helloworld# cat go.mod
module helloworld

go 1.14
root@b2d6e16e7e6e:/go/src/helloworld# go build
go: finding module for package helm.sh/helm/v3/pkg/chart/loader
go: found helm.sh/helm/v3/pkg/chart/loader in helm.sh/helm/v3 v3.2.4
go: downloading sigs.k8s.io/yaml v1.2.0
go: downloading github.com/pkg/errors v0.9.1
go: downloading gopkg.in/yaml.v2 v2.2.8
# helloworld
./main.go:4:2: imported and not used: "helm.sh/helm/v3/pkg/chart/loader"
```

More detail as to why this occurs is explained below.

I started by trying to reproduce this from a clean state, and was able to reproduce the same issue @plynch76 reported:

```
><> docker run -it golang:latest bash
root@b2d6e16e7e6e:/go# go get helm.sh/helm/v3/pkg/chart/loader
unrecognized import path "helm.sh/helm/v3/pkg/chart/loader": reading https://helm.sh/helm/v3/pkg/chart/loader?go-get=1: 404 Not Found
```

Fetching the root package works for me, however.

```
root@b2d6e16e7e6e:/go# go get helm.sh/helm/v3
package helm.sh/helm/v3: no Go files in /go/src/helm.sh/helm/v3
root@b2d6e16e7e6e:/go# ls src/helm.sh/helm/v3/
ADOPTERS.md      KEYS     Makefile  README.md    cmd                 go.mod  internal  scripts
CONTRIBUTING.md  LICENSE  OWNERS    SECURITY.md  code-of-conduct.md  go.sum  pkg       testdata
```

This is due to how https://helm.sh is set up. We have a URL set up for `https://helm.sh/helm/v3` but not for any subtrees under that URL path. See https://github.com/helm/helm-www/blob/master/content/code/helm-v3.md

You can inspect the same behaviour with `curl`:

```
root@b2d6e16e7e6e:/go# curl https://helm.sh/helm/v3
<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=go-import content="helm.sh/helm/v3 git https://github.com/helm/helm"><meta name=go-source content="helm.sh/helm/v3 git https://github.com/helm/helm https://github.com/helm/helm/tree/master{/dir} https://github.com/helm/helm/blob/master{/dir}/{file}#L{line}"><meta http-equiv=refresh content="0; url=https://github.com/helm/helm"></head><body>Nothing to see here; <a href=https://github.com/helm/helm>move along</a>.</body></html>
```

And for the URL @plynch76 is using:

```
root@b2d6e16e7e6e:/go# curl https://helm.sh/helm/v3/pkg/chart/loader -I
HTTP/2 404
cache-control: public, max-age=0, must-revalidate
content-type: text/html; charset=utf-8
date: Tue, 23 Jun 2020 17:42:00 GMT
etag: 1547678321-ssl
age: 7
content-length: 17261
server: Netlify
x-nf-request-id: ef3a93ef-683d-4668-930b-98b1e499078d-1520682
```

In other words, you cannot `go get` subpackages directly. You have to fetch `helm.sh/helm/v3` first before fetching subpackages. That or use go modules.

**Comment by plynch76** [comment]
Thanks all....converting my project to go modules seems to have fixed the
problem. :)
